float64 accelXInG
float64 accelYInG
float64 accelZInG
